.travis.yml

```diff
@@ -3,14 +3,11 @@ sudo: false
 cache: bundler
 branches:
   only: master
-before_install:
-  - gem update --system
-  - gem update bundler
 script: bundle exec rake $TASK
 env:
 - TASK=test
 - TASK=rubocop
 rvm:
   - 2.3
   - 2.4
-  - 2.5.0
+  - 2.5
```

Comment:
<img width="799" alt="screen shot 2018-04-19 at 3 27 22 pm" src="https://user-images.githubusercontent.com/22880578/39165294-599fb0dc-47b6-11e8-859e-17a54e3635a1.png">

Travis still has a bug with installing `2.5.1` then trying to load the gems from `2.5.0`